crates/storage/src/merkle/column.rs

```diff
@@ -56,7 +63,9 @@ where
             Self::MerkleDataColumn(column) => {
                 Self::MERKLE_DATA_COLUMNS_START.wrapping_add(column.as_u32())
             }
-            Self::MerkleMetadataColumn => u32::MAX,
+            Self::MerkleMetadataColumn => {
+                Self::COUNT.checked_sub(1).unwrap().try_into().unwrap()
+            }
```

Comment:
Hmm it could be worth to try if we could replace this with a compile-time evaluated expression. Don't know by heart how that would look, but we could get rid of the run-time unwraps that way.

Reply:
i think its overkill, but did it anyway - 50ad25626a

Reply:
Niiice! Looks great 🤩 I think this is significantly better. Although the unwraps are technically safe, it still introduces a (very small but non-zero) risk that future bugs accidentally trigger this panic during runtime, thus causing downtime. Now, that risk is instead eliminated.